src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter_Call.cs

```diff
@@ -486,7 +486,7 @@ void ensureTempTrackingSetup([NotNull] ref ArrayBuilder<LocalSymbol>? temps, ref
                                     Debug.Assert(visitedReceiver != null && requiresInstanceReceiver);
                                     local = _factory.StoreToTemp(visitedReceiver, out var store, refKind: visitedReceiver.GetRefKind());
                                     temps.Add(local.LocalSymbol);
-                                    visitedReceiver = _factory.Sequence(ImmutableArray<LocalSymbol>.Empty, ImmutableArray.Create<BoundExpression>(store), local);
+                                    visitedReceiver = _factory.Sequence(ImmutableArray<LocalSymbol>.Empty, ImmutableArray.Create<BoundExpression>(store), local, forceSpill: true);
```

Comment:
>forceSpill: true [](http://example.com/codeflow?start=161&length=16)

I am not comfortable with unconditionally setting this flag here. Especially that it might appear to someone that setting the flag on its own is going to trigger the spilling, whereas, it looks like ```_needsSpilling``` should also be set for the ```SpillSequenceSpiller``` to be invoked. I think that a better approach would be to understand why the```SpillSequenceSpiller``` doesn't do proper spilling and get it to do one. The issue feels very similar to the one fixed in [https://github.com/dotnet/roslyn/pull/58657](https://github.com/dotnet/roslyn/pull/58657). In there, ```SpillSequenceSpiller``` was explicitly trying to spill an expression containing an "interesting" sequence, but was not doing that properly, i.e. was failing to do the spilling for all "interesting" parts. While a conservative approach of always spilling this sequence addresses the issue at hand, I think it would be better to have ```SpillSequenceSpiller``` implementation on witch we can rely on in other similar scenarios.